community/server/src/main/java/org/neo4j/server/Bootstrapper.java

```diff
@@ -76,6 +77,7 @@ public static void main( String[] args )
     public int start( File configFile, Pair<String, String> ... configOverrides )
     {
         LogProvider userLogProvider = FormattedLogProvider.withoutRenderingContext().toOutputStream( System.out );
+        System.setOut( NullPrintStream.getInstance() );
```

Comment:
Fixing all of those, whenever they pop up, is difficult. New sources can appear at times due to such things as library upgrades, as in the case of 2.3.0 and netty 4.0.

Standard practice in server applications is to setup a logging system, then close stdout. That's why I feel comfortable with this approach.

(That said, our logging framework currently must write it's output to stdout. We can and should change that in 3.0 to write directly to a neo4j.log file.)